### DISCUSSION TITLE
deep-live-came for mac OS

### DISCUSSION THREAD

**Discussion by yuanxx333** [hacksider/Deep-Live-Cam#762] [Bounty Ideas]
i totally want  a deep-live-cam  for  mac OS !

**Comment by hacksider** [comment]
You can, just follow the instruction on the readme. Just an info, it will be very slow.

**Comment by KRSHH** [comment]
> i totally want a deep-live-cam for mac OS !

If you're talking about the prebuilt, Its not planned. atleast for now.
